src/statsig/constants.ts

```diff
@@ -65,6 +65,7 @@ export const DynamicConfigs = {
       maxSlippagePercentage: '0.3',
       enableAppFee: false,
       popularTokenIds: [] as string[],
+      enabled: false,
```

Comment:
What do you think of enabling it by default (and the app fee too)?
Seems like it would be a better default now that it's battle tested.

Reply:
i wondered this but thought it was safer since we're not allowed to enable this feature in some regions for compliance reasons?